README.md

```diff
@@ -0,0 +1,179 @@
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Rent A Car <a name="about-project"></a>
+
+**Rent A Car** A beautifull web app create using Ruby on Rails and React js where users can rent cars.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+   <li>HTML</li>
+   <li>CSS</li>
+   <li>Javascript</li>
+   <li>React Js</li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>Ruby on Rails</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>Postgresql</li>
+  </ul>
+</details>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- `N/A`
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Simple UI**
+- **Beautiful Design**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Setup
+
+Clone the repo into your local machine:
+
+```bash
+git clone https://github.com/BahirHakimy/rent-a-car.git
+```
+
+### Prerequisites
+
+In order to run this project you need to:
+
+- Install [Git](https://git-scm.com/)
+- Install [Ruby](https://rubyinstaller.org/)
+- Install [Rails](https://rubyonrails.org/)
+
+### Install
+
+To install the dependencies run the following command
+
+```bash
+bundle install
+npm install
+```
+
+### Usage
+
+To start the project execute this command
+
+```bash
+./bin/dev
+```
```

Comment:
- [ ] When  I run the project according to your instructions, I get this:

![image](https://github.com/BahirHakimy/rent-a-car/assets/57421823/459f7551-ac0a-4197-acd8-87835f0c013d)

1. Is tailwindcss included in package.json dependencies? If not, kindly include it so your users have all the dependencies.

2. Are there any more procedures required to run your project correctly other than those included on your readme?  If that's the case, kindly add the instructions to your readme.  Please, ensure any user can setup and run your project just by following your instructions.

_note: This means I wasn't able to check your project's functionality and styling.  This will be taken a look at the next review._